src/evaluator.rs

```diff
@@ -177,7 +177,12 @@ impl<'src, 'run> Evaluator<'src, 'run> {
       }
       Expression::Group { contents } => self.evaluate_expression(contents),
       Expression::Join { lhs, rhs } => {
-        Ok(self.evaluate_expression(lhs)? + "/" + &self.evaluate_expression(rhs)?)
+        let lhs = if let Some(lhs) = lhs {
+          self.evaluate_expression(lhs)?
+        } else {
+          String::new()
+        };
```

Comment:
Minor, feel free to ignore. You might consider `match` for this one:

```rust
        let lhs = match lhs {
            Some(lhs) => self.evaluate_expression(lhs)?,
            None => String::new(),
        };
```

Reply:
I am down with it 👍